vault/package/install.sls

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+{% from "vault/map.jinja" import vault with context %}
+
+vault-package-install-group-present:
+  group.present:
+    - name: vault
+    - system: True
+
+vault-package-install-user-present:
+  user.present:
+    - name: vault
+    - system: True
+    - gid_from_name: True
+    - home: /var/lib/vault
+
+vault-package-install-file-directory:
+  file.directory:
+    - name: /opt/vault/bin
+    - makedirs: True
+
+vault-package-install-archive-extracted:
+  archive.extracted:
+    - name: /opt/vault/bin
+    - source: https://releases.hashicorp.com/vault/{{ vault.version }}/vault_{{ vault.version }}_{{ vault.platform }}.zip
+    - source_hash: https://releases.hashicorp.com/vault/{{ vault.version }}/vault_{{ vault.version }}_SHA256SUMS
+    - source_hash_name: vault_{{ vault.version }}_{{ vault.platform }}.zip
+    - enforce_toplevel: false
```

Comment:
Refer to [this comment](https://github.com/saltstack-formulas/vault-formula/pull/23/files?file-filters%5B%5D=.sls&file-filters%5B%5D=.yaml&hide-deleted-files=true#r268416313).